src/askem_beaker/contexts/decapodes/context.py

```diff
@@ -224,6 +224,25 @@ async def save_amr_request(self, message):
             "iopub", "save_model_response", content, parent_header=message.header
         )
 
+    @intercept()
+    async def model_to_equation(self, message):
```

Comment:
typically intercepts end with `_request` for clarity

Reply:
done